installinstallmacos.py

```diff
@@ -262,7 +262,7 @@ def replicate_url(full_url,
         if attempt_resume:
             curl_cmd.extend(['-C', '-'])
     curl_cmd.append(full_url)
-    print("Downloading %s..." % full_url)
+    print("Downloading %s..." % full_url, file=sys.stderr)
```

Comment:
Hmmm. I know why you want this; I just have a philosophical issue with directing non-error messages to stderr. Convince me this is the right thing to do. Is this a common pattern for UNIX/POSIX tools?